forloop_modules/utils/various.py

```diff
@@ -29,26 +29,33 @@ def is_value_redis_compatible(value) -> bool:
     )
 
 
-def serialize_dataframe_to_api(value_df: pd.DataFrame) -> dict:
+def serialize_if_dataframe_to_api(variable_df: pd.Series) -> Any:
```

Comment:
very bad name for a method, its also not easy to understand here - variable_df is dataframe/series always I hope

Reply:
Explain this better + rename method
What is value_df? What is variable_value_df? make these things super clear

Reply:
When its series, why is it called DF? -> cast to Df ASAP or call it series